SWLOR.Game.Server/Service/PlayerStatService.cs

```diff
@@ -476,21 +476,25 @@ public static EffectiveItemStats GetPlayerItemEffectiveStats(NWPlayer player, NW
 
 
                 // Calculate AC
-                if (ItemService.ArmorBaseItemTypes.Contains(item.BaseItemType) ||
-                    ItemService.ShieldBaseItemTypes.Contains(item.BaseItemType))
+                if (ItemService.ArmorBaseItemTypes.Contains(item.BaseItemType))
```

Comment:
> According to my testing and math, it should be fine, right now combat is mainly a contest of natural 20's on people that would get affected by this. I still think the numbers may be too high, but i'd rather tweak it over time.

> How would you approach it? Because I know limiting AC gain items in game would be one way but that doesn't affect the legacy stuff. Which is what this mainly hits.

I need to understand the problem better first. Is it just legacy gear or is it stuff people can make today?

Reply:
Both Potentially. But Mainly legacy

Reply:
I really hate it, but I'll concede to you. We can try it out. Fix the other stuff though :P